client/Bootstrap/index.tsx

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+
+import '@/Styling';
```

Comment:
I know you've mentioned the `@/` style before, is there a reason for the `/`? I've seen some examples where it's just `@module` for local aliases which blends in nicely with some `node_modules` imports

Reply:
So I was thinking of it more as a convention for "local" modules - as lots of node modules use `@module` for namespacing. Happy to remove the `/` though if we want to look more like all other modules.

Reply:
I dont mind the `/` or `@/` myself, as long as the `@/` is documented as being 'local' code